specification/2.0/README.md

```diff
@@ -2183,38 +2150,64 @@ Application-specific data.
 <a name="reference-material"></a>
 ## material
 
-The material appearance of a primitive.
-
-**Properties**
-
-|   |Type|Description|Required|
-|---|----|-----------|--------|
-|**technique**|`string`|The ID of the [`technique`](#reference-technique).|No|
-|**values**|`object`|A dictionary object of parameter values.|No, default: `{}`|
-|**name**|`string`|The user-defined name of this object.|No|
-|**extensions**|`object`|Dictionary object with extension-specific objects.|No|
-|**extras**|`any`|Application-specific data.|No|
-
-Additional properties are not allowed.
-
+glTF 2.0 uses the metal-roughness material model from Physically Based Renderring. The metal-roughness material model is defined by the following properties:
+* Base Color: The base color of the material
+* Metallic-ness: A value between 0 and 1, indicating if the material is metallic (1) or not (0)
+* Roughness: Roughness of the material surface
+
+Todo: Need a good figure illustrating this
+
+The `base` color has two different interpretations depending on the value of `metallic`, which is defined as `0` for dielectrics and `1` for metals.
+For `metallic = 1`, `base` is the specific measured reflectance value (F0).
+For `metallic = 0`, `base` represents the reflected diffuse color of the material.
+
+In this model it is not possible to specify a reflectance value for non-metals, where a reflectance value of 4% (0.04) is often used.
+The `roughness` property is related with the `glossiness` parameter in the specular-glossiness model and is defined as `roughness = 1 - glossiness`. 
+
```

Comment:
Although it is shortly *mentioned* in the "meshes" section, the term **"set index"** is used several times here, but it might not be clear for everybody what this refers to. Elaborating it here (or detailing it at another point and adding a link) may be helpful.